nnpdf_data/nnpdf_data/new_commondata/STAR_2013_2JET_C_510GEV/uncertainties.yaml

```diff
@@ -0,0 +1,1042 @@
+    treatment: ADD
+    type: STAR2013JETunc39
+  sys_40:
+    description: 40 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc40
+  sys_41:
+    description: 41 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc41
+  sys_42:
+    description: 42 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc42
+  sys_43:
+    description: 43 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc43
+  sys_44:
+    description: 44 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc44
+  sys_45:
+    description: 45 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc45
+  sys_46:
+    description: 46 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc46
+  sys_47:
+    description: 47 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc47
+  sys_48:
+    description: 48 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc48
+  sys_49:
+    description: 49 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc49
+  sys_50:
+    description: 50 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc50
+  sys_51:
+    description: 51 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc51
+  sys_52:
+    description: 52 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc52
+  sys_53:
+    description: 53 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc53
+  sys_54:
+    description: 54 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc54
+  sys_55:
+    description: 55 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc55
+  sys_56:
+    description: 56 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc56
+  sys_57:
+    description: 57 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc57
+  sys_58:
+    description: 58 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc58
+  sys_59:
+    description: 59 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc59
+  sys_60:
+    description: 60 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc60
+  sys_61:
+    description: 61 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc61
+  sys_62:
+    description: 62 artificial correlated systematics uncertainty
+    treatment: ADD
+    type: STAR2013JETunc62
```

Comment:
In my opinion, the systematic uncertainty (`ADD`, `MULT`) due to the shift of the luminosity uncertainty (4.7*10^-4) is missing.